react/components/molecules/snackbar-ios/snackbar-ios.js

```diff
@@ -0,0 +1,125 @@
+import React, { PureComponent } from "react";
+import { Animated, Platform, StyleSheet, Text, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { baseStyles, IdentifiableMixin } from "../../../util";
+
+import { Touchable } from "../../atoms";
+
+export class SnackbarIos extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            text: PropTypes.string,
+            actionText: PropTypes.string,
+            duration: PropTypes.number,
+            animationDuration: PropTypes.number,
+            onActionPress: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            text: undefined,
+            actionText: undefined,
+            duration: 5000,
+            animationDuration: 300,
+            onActionPress: () => {},
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            opacity: new Animated.Value(0),
+            snackbarTimeout: null
+        };
+    }
+
+    _stopPrevAnimations() {

```

Comment:
Or else the `this` keyword will be undefined, since the function is not binded to it
```suggestion
    _stopPrevAnimations = () => {

```